### PR TITLE
Expose sign op 

### LINKED ISSUE #17116: [Operator] add sign op
sign op implements the same function as the numpy API np.sign.

### PR DESCRIPTION
**Pull request by zhoukunsheng**
fix #17116 
This commit exposes the already existed sign kernel to public.

### CODE REVIEW

**Comment by zhoukunsheng** [comment]
image

**Review by phlrain — DISMISSED** [review]
LGTM

**Review by shanyi15 — DISMISSED** [review]
LG comments
It's better to amplify the explanation of Return part.

File: python/paddle/fluid/layers/nn.py [hunk 1]
```diff
@@ -11267,3 +11268,35 @@ def continuous_value_model(input, cvm, use_cvm=True):
...
+    helper = LayerHelper("sign", **locals())
+
+    if not isinstance(x, Variable):
+        x = assign(x)
+
+    out = helper.create_variable_for_type_inference(dtype=x.dtype)
+
+    helper.append_op(type='sign', inputs={'X': [x]}, outputs={'Out': [out]})
+
+    out.stop_gradient = True
```
Comment (chengduoZH) [inline python/paddle/fluid/layers/nn.py]: Why do you add `out.stop_gradient = True` here?

Comment (zhoukunsheng) [reply python/paddle/fluid/layers/nn.py]: ok, let's let the user to decide whether they need gradient or not.

**Review by chengduoZH — DISMISSED** [review]

**Review by shanyi15 — DISMISSED** [review]

**Comment by luotao1** [comment]
@zhoukunsheng please solve the conflicts.

**Review by shanyi15 — APPROVED** [review]

**Review by chengduoZH — APPROVED** [review]

**Pull request merged**